packages/core/content-manager/admin/src/containers/EditSettingsView/LinkToCTB/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ *
+ * EditViewButton
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useGlobalContext, CheckPermissions } from '@strapi/helper-plugin';
+import { useHistory } from 'react-router-dom';
+import get from 'lodash/get';
+import { Button } from '@buffetjs/core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import getTrad from '../../../utils/getTrad';
+
+const permissions = [{ action: 'plugins::content-type-builder.read', subject: null }];
+
+// Create link from content-type-builder to content-manager
+const LinkToCTB = ({ modifiedData, slug, type }) => {
```

Comment:
We don't need modifiedData here, I think you could simply pass the category instead :)